.github/workflows/ci.yml

```diff
@@ -144,18 +141,3 @@ jobs:
         with:
           command: tarpaulin
           args: --out xml
-
-  publish:
```

Comment:
We'll probably want to bring this back, but probably in a separate action triggered by a tag push / release creation.

As it is all the publish logic is missing anyway.